app/assets/stylesheets/main.scss

```diff
@@ -98,6 +30,10 @@ form .hint {
   max-height: 100px;
 }
 
+.sponsor-sm {
+  max-width: 120px;
```

Comment:
is this not a class in bootstrap?

Reply:
@KimberleyCook unfortunately no, would be with Tailwind but not Bootstrap.